modules/sfp_dnsraw.py

```diff
@@ -144,6 +144,14 @@ def handleEvent(self, event):
                                         evt = SpiderFootEvent(evt_type, domain, self.__name__, parentEvent)
                                         self.notifyListeners(evt)
 
+                                        if self.sf.isDomain(domain, self.opts['_internettlds']):
+                                            if evt_type.startswith('AFFILIATE'):
+                                                evt = SpiderFootEvent('AFFILIATE_DOMAIN_NAME', domain, self.__name__, event)
+                                                self.notifyListeners(evt)
+                                            else:
+                                                evt = SpiderFootEvent('DOMAIN_NAME', domain, self.__name__, event)
```

Comment:
This will result in recursive behavior right? Because the module is now generating `DOMAIN_NAME`, which it's also listening for.

Reply:
Fixed